catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsLoaderCommand.java

```diff
@@ -0,0 +1,123 @@
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.catalog.ui.forms.SearchFormsLoader;
+import org.codice.ddf.catalog.ui.forms.SearchFormsUtil;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
+import org.codice.ddf.commands.catalog.SubjectCommands;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Service
+@Command(
+  scope = "forms",
+  name = "load",
+  description = "Handles the ingestion of search forms and result set forms into the system"
+)
+public class SearchFormsLoaderCommand extends SubjectCommands {
+  private static final Logger LOGGER = LoggerFactory.getLogger(SearchFormsLoaderCommand.class);
+
+  @Reference CatalogFramework catalogFramework;
+
+  @Reference EndpointUtil endpointUtil;
+
+  @Option(
+    name = "--formsDirectory",
+    required = false,
+    aliases = {"-d"},
+    multiValued = false,
+    description = "Overrides the default forms directory path. Default: /etc/forms"
+  )
+  protected String formsDir = null;
+
+  @Option(
+    name = "--forms",
+    required = false,
+    aliases = {"-f"},
+    multiValued = false,
+    description = "Override the default forms.json file name. Default: forms.json"
+  )
+  protected String formsFile = null;
+
+  @Option(
+    name = "--results",
+    required = false,
+    aliases = {"-r"},
+    multiValued = false,
+    description = "Override the default results.json file name. Default: results.json"
+  )
+  protected String resultsFile = null;
+
+  @Override
+  protected Object executeWithSubject() {
+    final PrintStream console = System.out;
+    final SearchFormsLoader loader;
+
+    console.println(ansi().fgBrightCyan().a("Initializing Search Form Template Loader").reset());
+
+    if (SearchFormsUtil.allNull(formsDir, formsFile, resultsFile)) {
+      console.println(
+          ansi()
+              .fgYellow()
+              .a("No options selected, falling back to default directory locations.")
+              .reset());
+      loader = generateLoader(catalogFramework, endpointUtil, null, null, null);
+    } else {
+      LOGGER.debug("Attempting loader initialization");
+      loader = generateLoader(catalogFramework, endpointUtil, formsDir, formsFile, resultsFile);
+    }
+
+    List<Metacard> parsedMetacards = loader.retrieveSystemTemplateMetacards();
+
+    if (!parsedMetacards.isEmpty()) {
+      console.println(
+          ansi()
+              .fgYellow()
+              .a("Loader initialized, beginning ingestion of system templates.")
+              .reset());
+      loader.bootstrap(parsedMetacards);
+      console.println(ansi().fgBrightGreen().a("System templates successfully ingested.").reset());
+      return null;
+    }
+
+    console.println(ansi().fgBrightRed().a("No system forms to load, halting ingest.").reset());
+    return null;
+  }
+
+  @VisibleForTesting
+  public SearchFormsLoader generateLoader(
+      CatalogFramework cf,
```

Comment:
I think in our code style we generally stay away from shortened variable names like this.

Reply:
I've found us to be pretty inconsistent with this across the code base and in fact we have a lot of these shortened variable names everywhere. `catalogFramework` seems to be our most popular choice for this one though.